usl_pipeline/usl_lib/usl_lib/readers/polygon_readers.py

```diff
@@ -0,0 +1,67 @@
+import typing
+
+import fiona
+import fiona.crs
+import pyproj
+from shapely.geometry import Polygon
+
+
+SHAPE_FILE_POLYGON_FEATURE_TYPE = "Polygon"
+
+
+def read_polygons_from_shape_file(
+    file_path: str,
+    target_crs: typing.Optional[str] = None,
+    mask_value_feature_property: typing.Optional[str] = None,
+    skip_zero_mask_values: bool = True,
+) -> list[typing.Tuple[Polygon, int]]:
+    """Reads polygon data from shape file.
+
+    Function reads all the polygon shapes from shape file. There is optional support for
+    transformation of coordinates to different CRS. Another option is to load masks
```

Comment:
When you write 'Another option is' that read more like, 'An alternative option is,' as in, an alternative to the first option rather than a second, additional optional parameter. I'd write this as, 

"The function supports several optional transformations. `target_crs` transforms the coordinates to a different CRS. `mask_value_feature_property` loads masks ...

Or alternatively, just rely on the below 'Args:' section to describe these.

Reply:
Done